packages/perseus/src/widgets/radio/__tests__/multiple-choice.test.ts

```diff
@@ -26,7 +26,7 @@ import type {UserEvent} from "@testing-library/user-event";
 const createApiOptions = (options: Partial<APIOptions> = {}): APIOptions => ({
```

Comment:
if we are only setting the APIOption with the same values, we can simplify the test by using the `beforeEach` block:

```suggestion
let apiOptions: APIOptions;
beforeEach(() => {
    // Reset the API options before each test
    apiOptions = createApiOptions();
});
// Create API options with the new-radio-widget flag enabled
const createApiOptions = (options: Partial<APIOptions> = {}): APIOptions => ({
    ...options,
    flags: {
        "new-radio-widget": false,
        ...(options.flags || {}),
    },
});
```

then in each test we can remove this line:
const apiOptions = createApiOptions();